themes/beaver/layouts/partials/blog/img-cropped.html

```diff
@@ -0,0 +1,15 @@
+{{ $image := .Page.Resources.GetMatch (printf "*%s*" .Page.Params.metatags.image) }}
+{{ if $image }}
+{{ $cropped := $image.Fill (printf "%dx%d Center" .width .height) }}
+{{ $webp := $cropped.Resize (printf "%dx%d webp" .width .height) }}
+
+
+<picture>
+    <source srcset="{{ $webp.Permalink }}" type="image/webp">
+    <source srcset="{{ $cropped.Permalink }}" type="{{ $cropped.MediaType }}">
+    <img src="{{ $cropped.Permalink }}" alt="{{ .Page.Title }}" width="{{ .width }}" height="{{ .height }}">
+</picture>
+
+{{ else }}
+<img src="/img/no-image.svg" alt="Placeholder Image" width="{{ .width }}" height="{{ .height }}">
```

Comment:
do not use hard coded static path, use processed path or better inline svg

Reply:
we don't need to process svg
we don't need to inline svg if that svg will be used 100 times

Reply:
![nDEU72hACa](https://github.com/user-attachments/assets/760462d0-23b2-40a7-a5f0-be752bab2c60)

Reply:
++